packages/spor-input-react/src/ChoiceChip.tsx

```diff
@@ -17,6 +17,7 @@ export type ChoiceChipProps = {
   icon?: React.ReactNode;
   size?: "sm" | "md" | "lg" | "xl";
   variant?: "icon" | "choice" | "filter";
+  checkedIcon?: React.ReactNode;
```

Comment:
Dette fungerer, og er ikke breaking – så kanskje vi rett og slett skal kjøre på med denne. Vi kan vurdere å endre APIet til dette i fremtiden, for å sikre at folk sender inn begge deler:

```
icon?: {
  default: React.ReactNode;
  checked: React.ReactNode;
}
```
, for da blir du tvunget til å sende inn begge deler, om du først skal sende inn et ikon. Men det er en breaking change, så jeg tenker at vi innfører det i v1